roles/horizon/tasks/main.yml

```diff
@@ -42,6 +42,21 @@
   notify:
     - reload apache
 
+- name: set horizon keystone policy fact
+  set_fact: policy_for_horizon=True
+  when: horizon.customize|default('False')|bool
+
+# Use custom policy to allow cloud_admin admin panel access
+- name: configure custom keystone policy
+  template:
+    src: "roles/keystone/templates/etc/keystone/policy.json"
+    dest: "/etc/openstack-dashboard/keystone_policy.json"
+    mode: 0644
```

Comment:
The current policy files were required to have 644 because we wanted horizon to be able to read them. Ideally, I would like to create a horizon policy file for each of the services as well and make the permissions 640 on all the policy files. None of these should be world readable IMHO.